quinn/examples/server.rs

```diff
@@ -87,7 +87,9 @@ async fn run(options: Opt) -> Result<()> {
         };
         let cert_chain = fs::read(cert_path).context("failed to read certificate chain")?;
         let cert_chain = if cert_path.extension().map_or(false, |x| x == "der") {
-            quinn::CertificateChain::from_certs(quinn::Certificate::from_der(&cert_chain))
+            quinn::CertificateChain::from_certs(Some(
```

Comment:
is that intentional?

Reply:
Yes, see the last commit message; just some cleanup I threw in after being briefly confused while referencing the example.

Reply:
Ah ok. I wasn't even aware that this is a `Result`. I thought the method returns an `Option`, in which case this piece of code would just have unwrapped and re-wrapped it.

Reply:
Yeah, it was pretty confusing, hence the fix :D